apps/dashboard/src/tw-components/AddressCopyButton.tsx

```diff
@@ -70,14 +70,9 @@ export const AddressCopyButton: React.FC<AddressCopyButtonProps> = ({
           e.stopPropagation();
           e.preventDefault();
           onCopy();
-          toast({
-            variant: "solid",
-            position: "bottom",
-            title: `${title.charAt(0).toUpperCase() + title.slice(1)} copied.`,
-            status: "success",
-            duration: 5000,
-            isClosable: true,
-          });
+          toast.success(
```

Comment:
This shouldn't be "success" variant - if there is other visual feedback for copy action - we can completely drop this - otherwise use "info" variant

Also, this entire component needs to be removed and replaced with <WalletAddress />